### ISSUE TITLE
Suspected memory leak

### ISSUE THREAD

**Issue by WhAtEvErYoUmEaN** [LibreHardwareMonitor/LibreHardwareMonitor#504]
I haven't put any effort into debugging it, but if you leave LibreHardwareMonitor running (using the widget in my case) the allocated virtual memory keeps climbing up into multiple gigabytes

**Comment by nangusa** [comment]
Take a look at this: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/450

**Comment by WhAtEvErYoUmEaN** [comment]
@nangusa so you're saying this is either the nVidia driver or the sensor value history? Did i get that correct?
Pardon the question, but i don't see any option to turn the latter off.

I resented to launching LibreHardwareMonitor via the task scheduler and killing/restarting the instance every so often as a workaround in the meantime

**Comment by nangusa** [comment]
@WhAtEvErYoUmEaN Yes, that's what I suspect it's your memory leak. In LHM, you can navigate to "Options/Sensor Values Time Window" and set it to 30 seconds to minimize the memory footprint.

If you have an nVidia card, update the drivers to any 47x.xx version.

**Comment by 1usmus** [comment]
> @WhAtEvErYoUmEaN Yes, that's what I suspect it's your memory leak. In LHM, you can navigate to "Options/Sensor Values Time Window" and set it to 30 seconds to minimize the memory footprint.
> 
> If you have an nVidia card, update the drivers to any 47x.xx version.

**Sensor.cs** 
private TimeSpan _valuesTimeWindow = TimeSpan.Zero;

Unfortunately, even this does not solve the problem. AMD graphics cards...
